documents/motions/10.2024.4 - Appeals Committee.md

```diff
@@ -0,0 +1,33 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 10.2024.4  
+**Subject:** WCA Appeals Committee  
+**Intent:** Rights and duties of the WCA Appeals Committee  
+**Submitted by:** WCA Board  
+**Date:** TBC, 2024  
+
+# Motion
+
+The WCA Appeals Committee is an Advisory Committee of the WCA.
+
+1. The WCA Appeals Committee has the following rights and duties:
+   1. Forming the Board of Appeal to carry out the appeals processes described in [WCA Motion : Disputes](wca{documents/motions/16.2021.1 - Disputes.pdf}).
+      1. The Board of Appeal shall be made up of three members of the WCA Appeals Committee.
+      2. The Leader of the WCA Appeals Committee determines the members of the Board of Appeal for each dispute.
+      3. Members of the WCA Appeals Committee who have a conflict of interest for a specific dispute shall be ineligible from selection to that specific Board of Appeal.
+      4. The Board of appeal shall have all the rights and duties assigned to the WCA Appeals Committee and shall have the full right to represent the committee in the appeals process.
+   2. Performing independent investigations regarding appealed incidents.
+   3. Making decisions on disciplinary actions in relation to appealed incidents.
+   4. Informing the WCA Board on appeals, investigations, decisions, and disciplinary actions.
+   5. Publishing public information on the appeals, investigations, decisions, and disciplinary actions where necessary.
+   6. Managing an administration of the appeals, investigations, the reports, the decisions, and the disciplinary actions.
+   7. Additional duties as directed by the WCA Board.
+2. All WCA Staff and Registered Speedcubers must fully cooperate with investigations by the WCA Appeals Committee.
+3. In case the WCA Appeals Committee is not constituted or for any reason non-functional, the WCA Board may appoint select WCA Staff to handle cases of appeal.
+4. The WCA Appeals Committee will take the role of WCA Integrity Committee in case the WCA Integrity Committee is not constituted or for any reason non-functional.
+      1. For any period where the WCA Appeals Committee assumes the role of the WCA Integrity Committee the WCA Board may appoint select WCA Staff to handle cases of appeal.
+5. Reducing conflicts of roles:
+   1. If all bodies are constituted and functional there must not exist an overlap in members of the WCA Board, WCA Integrity Committee, and WCA Appeals Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader.
+6. Eligibility for membership:
+   1. To be eligble to serve as a member of the WCA Appeals Committee the candidate must have served a period of at least 12 consecutive months as a member of WCA Staff.
```

Comment:
Does it matter when that experience is from?  Should it be recent experience or does it not matter?  Personally I don't think it matters all that much but it is worth considering

Reply:
I have left it without restriction for now just to indicate that 12 months is required but not where specifically. I think this gives the leader some flexibility while noting this position is not for members who have little to no idea on the WCA's internal structure.